src/pool.jl

```diff
@@ -184,7 +191,45 @@ function merge_pools(a::CategoricalPool{T}, b::CategoricalPool) where {T}
     newlevs, ordered
 end
 
-Base.issubset(a::CategoricalPool, b::CategoricalPool) = issubset(a.levels, keys(b.invindex))
+Base.hash(pool::CategoricalPool, h::UInt) = pool.hash
```

Comment:
I would rather write `hash(pool.hash, h)` to have `hash` be sensitive to `h`.

Reply:
Woops indeed that's a mistake.